### PR TITLE
Fix issue with SupressWarnings

### PR DESCRIPTION
**Pull request by lyriccoder**
fix https://github.com/cqfn/aibolit/issues/585

### CODE REVIEW

File: test/recommend/start_end/FileStorage.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
+/*
```
Comment (acheshkov) [inline test/recommend/start_end/FileStorage.java:1]: add disclaimer here

File: aibolit/__main__.py [hunk 2]
```diff
@@ -242,12 +242,20 @@ def traverse(node):
...
-                        traverse(item)
+                        if isinstance(item, list):
+                            for i in item:
+                                traverse(i)
+                        else:
+                            traverse(item)
                 else:
                     if hasattr(child, 'children'):
                         infants = child.children
                         for infant in infants:
```
Comment (acheshkov) [inline aibolit/__main__.py:253]: why infants?

Comment (lyriccoder) [reply aibolit/__main__.py:253]: children of children

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@rultor release, tag=`1.2.6rc1`

**Pull request merged**